stl/inc/vector

```diff
@@ -441,6 +441,11 @@ private:
     friend class _Vb_val;
     friend _Tidy_guard<vector>;
 
+#if _HAS_CXX23
+    template <class>
+    friend class basic_stacktrace;
+#endif // _HAS_CXX23

```

Comment:
I think we need some comments about why to make stacktrace friend of it when the implementation settle down.